tests/unit/workflows/test_workflows_tasks_matching.py

```diff
@@ -65,11 +70,11 @@ def test_article_exists_returns_false_if_nothing_matched(mock_match):
     obj = MockObj(data, extra_data)
     eng = MockEng()
 
-    assert not article_exists(obj, eng)
-    assert 'record_matches' in obj.extra_data
+    assert not exact_match(obj, eng)
+    assert 'matches' in obj.extra_data
```

Comment:
It feels strange to me that you are checking that `'matches'` is in `extra_data` but not `'exact'`.

Reply:
there is this assertion after that is checking it, isn't it ok?
`result = get_value(obj.extra_data, 'matches.exact')`